### PR TITLE
[GH Pages] Update custom op documentation

### PR DESCRIPTION
**Pull request by adrianlizarraga**
### Description
Updates custom op documentation on the github pages site. Also, includes information about the [recent custom op APIs](https://github.com/microsoft/onnxruntime/pull/13427) that enable wrapping models to be inferenced with an external runtime.

Link to forked github pages with changes: https://adrianlizarraga.github.io/onnxruntime/docs/reference/operators/add-custom-op.html

### Motivation and Context
Add documentation for new custom op features in the next release.

### CODE REVIEW

File: docs/reference/operators/add-custom-op.md [hunk 1]
```diff
@@ -52,6 +216,94 @@ When using CUDA custom ops, to ensure synchronization between ORT's CUDA kernels
 
 For example, see how the afore-mentioned `MyCustomOp` is being launched and how the Session using this custom op is created.
 
+## Wrapping an external inference runtime in a custom operator
+A custom operator can wrap an entire model that is then inferenced with an external API or runtime. This can facilitate the integration of external inference engines or APIs with ONNX Runtime.
+
+As an example, consider the following ONNX model with a custom operator named "OpenVINO_Wrapper". The "OpenVINO_Wrapper" node encapsulates an entire MNIST model in OpenVINO's native model format (XML and BIN data). The model data is serialized into the node's attributes and later retrieved by the custom operator's kernel to build an in-memory representation of the model and run inference with OpenVINO C++ APIs.
+
+<p align="center"><img width="50%" src="https://www.onnxruntime.ai/images/custom_op_wrapper.png" alt="ONNX model of a custom operator wrapping an OpenVINO MNIST model"/></p>
```
Comment (adrianlizarraga) [inline docs/reference/operators/add-custom-op.md:224]: I added this image to the image/ folder. What is the proper way to get this image to show up in my personal github page?

Comment (faxu) [reply docs/reference/operators/add-custom-op.md:224]: Use relative urls

File: docs/reference/operators/add-custom-op.md [hunk 2]
```diff
@@ -52,6 +216,94 @@ When using CUDA custom ops, to ensure synchronization between ORT's CUDA kernels
 
 For example, see how the afore-mentioned `MyCustomOp` is being launched and how the Session using this custom op is created.
 
+## Wrapping an external inference runtime in a custom operator
+A custom operator can wrap an entire model that is then inferenced with an external API or runtime. This can facilitate the integration of external inference engines or APIs with ONNX Runtime.
+
+As an example, consider the following ONNX model with a custom operator named "OpenVINO_Wrapper". The "OpenVINO_Wrapper" node encapsulates an entire MNIST model in OpenVINO's native model format (XML and BIN data). The model data is serialized into the node's attributes and later retrieved by the custom operator's kernel to build an in-memory representation of the model and run inference with OpenVINO C++ APIs.
+
+<p align="center"><img width="50%" src="/images/custom_op_wrapper.png" alt="ONNX model of a custom operator wrapping an OpenVINO MNIST model"/></p>
```
Comment (faxu) [inline docs/reference/operators/add-custom-op.md:224]: I think this needs to be "../../../images/custom_op_wrapper.png"

Comment (adrianlizarraga) [reply docs/reference/operators/add-custom-op.md]: Updated

**Review by faxu — APPROVED** [review]

**Pull request merged**